dashboard-app/src/dashboard/ByVolunteer/VolunteerTabs.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FunctionComponent } from 'react';
+import styled from 'styled-components';
+import { Row, Col } from 'react-flexbox-grid';
+
+import _DataTable from '../../components/DataTable';
+import { aggregatedToStackedGraph } from '../dataManipulation/aggregatedToGraphData';
+import { TabGroup } from '../../components/Tabs';
+import { AggregatedData } from '../dataManipulation/logsToAggregatedData';
+import { DurationUnitEnum } from '../../types';
+import { TableData } from '../dataManipulation/aggregatedToTableData';
+import StackedBarChart from '../../components/StackedBarChart/index';
+
+/*
+ * Types
+ */
+
+interface Props {
+  aggData: AggregatedData;
+  unit: DurationUnitEnum;
+  tableData: TableData;
+  sortBy: number;
+  onChangeSortBy: (x: string) => void;
+  TITLE: string;
+}
+
+/*
+ * Styles
+ */
+const DataTable = styled(_DataTable)`
+  margin-top: 4rem;
+`;
+
+/*
+ * Component
+ */
+const VolunteerTabs: FunctionComponent<Props> = (props) => {
+  const { aggData, unit, tableData, sortBy, onChangeSortBy, TITLE } = props;
+
+  return(
+  <TabGroup titles={['Chart', 'Table']}>
```

Comment:
Indentation is all over the place in these files, we should fix the linting at some point in the future.